### ISSUE TITLE
Render man_made=water_well

### ISSUE THREAD

**Issue by zdila** [gravitystorm/openstreetmap-carto#1224]
Many famous mineral water sources are tagged as man_made=water_well so they should be rendered on OSM.

**Comment by mkoniecz** [comment]
Can you give examples?

**Comment by zdila** [comment]
http://www.sazp.sk/slovak/struktura/ceev/DPZ/pramene/vt/hn-4.html (there is a manual pump inside a shelter, mineral warer)
http://www.sazp.sk/slovak/struktura/ceev/DPZ/pramene/po/pv-12.html (mineral water well)
http://www.freemap.sk/upload/gallery/0/388/20101109154829_nkuijhcbu6njsdsk5nf9if11s5.jpg (mineral water)

**Comment by mkoniecz** [comment]
Sorry for being imprecise - can you give examples of OSM elements?

**Comment by zdila** [comment]
http://www.openstreetmap.org/node/3286489071

**Comment by maraf24** [comment]
Add amenity=drinking_water to them.

**Comment by zdila** [comment]
maraf24, I think this would be mapping for renderer. Maybe man_made=well could imply amenity=drinking_water unless it has drinking_water=no. Same for springs.

I would also like to have different icons for mineral water sources, but for that there is no standardised tagging yet.

**Comment by maraf24** [comment]
I was merely stating the fact that your water wells lack tag for drinkable water. It could be drinking_water=yes or amenity=drinking_water. The latter has an advantage of being rendered.

However the main point is that man_made=water_well is not rendered at all. Certainly, if it has drinking_water=yes it should be rendered at the same zoom level as amenity=drinking_water, maybe even with the same icon.
The same goes for recently approved man_made=water_tap.

**Comment by zdila** [comment]
Now what about springs and water_wells with dirty - non drinkable water? Should they be rendered?

Hypotetically somebody coult even tag water source as `amenity=drinking_water` together with `drinking_water=no`. It could be some amenity really built to provide drinking water which has become (maybe temporailry) contaminated.

**Comment by mkoniecz** [comment]
> Certainly, if it has drinking_water=yes it should be rendered at the same zoom level as amenity=drinking_water, maybe even with the same icon.

If it fits definition of amenity=drinking_water - it should be tagged also as amenity=drinking_water (it sounds like a good validator rule for JOSM). Also, drinking_water is currently not available in database.

> water_wells with dirty - non drinkable water? Should they be rendered?

IMHO rendering it at general purpose map is a poor idea. First problem is that it would be hard to express that it is water well with dirty water, not fully usable water well. Second problem - this feature is of really limited importance - even for me it is borderline.

**Comment by math1985** [comment]
As far as I'm concerned, it could be rendered on the highest zoomlevels as part of the mapper feedback loop. From z19, or maybe from z19.

**Comment by dieterdreist** [comment]
2015-01-19 11:46 GMT+01:00 maraf24 <notifications@github.com>:

> However the main point is that man_made=water_well is not rendered at all.
> Certainly, if it has drinking_water=yes it should be rendered at the same
> zoom level as amenity=drinking_water, maybe even with the same icon.



can also be interesting for historic wells, I have encountered quite a few
when mapping historic castles, mansions and the like. Sometimes they are
impressive structures, and they are nice to play for kids because of the
echo (they are usually secured by steel grates nowadays)

**Comment by Tomasz-W** [comment]
Can we get back to this topic? 

Now, there is over 81k of water wells. It's one of the most popular man_made=* values.
https://taginfo.openstreetmap.org/tags/?key=man_made&value=water_well

There is two options of potential icon shape:
-medieval well
https://i.pinimg.com/736x/a3/d7/db/a3d7dbb1eb1ca789debc8483ffaff993--water-well-zen-gardens.jpg
-modern pump
https://1.bp.blogspot.com/-wkrlDkQOq-U/UoJooKz_7VI/AAAAAAAAAVA/O8fDLx_PKok/s1600/SDC14667.JPG

What do you think? For me, a "pump" shape would be more universal and better.

**Comment by matkoniecz** [comment]
Note that icon must work in just 14 X 14 pixels.

**Comment by SomeoneElseOSM** [comment]
Here's what I went with in a different style: https://github.com/SomeoneElseOSM/openstreetmap-carto-AJT/blob/master/symbols/water_well.png .  If you want to see the effect on a map find a well you're interested in in UK or Ireland and then have a look at it on https://map.atownsend.org.uk/maps/map/map.html .

**Comment by Tomasz-W** [comment]
![icon_well2](https://user-images.githubusercontent.com/25656654/34640361-8189b224-f2f1-11e7-9bd8-98fc621f0915.png)

14x14: ![icon_well](https://user-images.githubusercontent.com/25656654/34640352-5ae2f428-f2f1-11e7-8a2c-63a3d079303e.png)

What are your thoughts? Is it readable enough?

**Comment by kocio-pl** [comment]
I like this shape in general - it's very clear for me even with 14 px matrix. It lacks solid bottom, water output should be L-shaped and be probably attached a bit higher, but these are just details.

I think that pump symbol should be used only for `man_made=water_well + pump=powered/manual`(~11k). For other wells I'd like to show what you called a "medieval well". What do you think about it?

**Comment by Tomasz-W** [comment]
Updated pump vector:

![icon_well2](https://user-images.githubusercontent.com/25656654/34648358-bdade728-f398-11e7-85fe-98c0f62fc0ed.png)

14x14: ![icon_well](https://user-images.githubusercontent.com/25656654/34648382-ef5e1d1a-f398-11e7-915a-e7b42557a755.png)

Gist: https://gist.github.com/Tomasz-W/b55760c0b5db057e2b6046cc3935cb21

Medieval well icon project:

![icon_well medieval 2](https://user-images.githubusercontent.com/25656654/34648446-70d99158-f399-11e7-8b31-f68bf9a22657.png)

14x14: ![icon_well medieval](https://user-images.githubusercontent.com/25656654/34648448-74e052aa-f399-11e7-858b-10540b2ba8b1.png)

I'm not sure about making two icons. I'm worried that many of pumps are tagged without "pump=powered/manual" tag, and it doesn't mean these objects have a "medieval well" shape.

What about the colour? I would like to see a test rendering in two versions - man_made grey, and water blue (like fountains)

**Comment by kocio-pl** [comment]
Great, that's really promising!

- I think pump bottom should be bigger, the output longer and with a curve, plus it could be attached higher. On 14 px it's mostly fine, but this time I'd like to tweak the general look.

- The well shape is good for me, I would only add a "rope" and I hesitate if the roof is really needed. I would also try if the wrench arm could be longer (this means probably shorter axis).

I guess that split is OK - if this is really the pump, one needs just to add the tag and plenty of images show me that traditional well is still popular.

I would use brown - man made is about things not usable for general public (just visible) and brown is for amenities, including `drinking_water`, which is very similar kind of object. I was trying different colors for fountains and I remember that marine blue was special, as other colors were simply too dark on water circle, so too much attention was given to the fountains, but that was not what I planned originally.

**Comment by Tomasz-W** [comment]
@kocio-pl I've put both svg files on Gist. Can you do this final changes on your own hand? You are more experienced, so you know better which sizes of certain elements would work fine. For me, it will be a guesswork ;)

**Comment by kocio-pl** [comment]
We have also `man_made=water_pump` ([760](https://taginfo.openstreetmap.org/tags/man_made=water_pump)) - I bet there's not always well, it can also be attached to a city water pipes for example.

**Comment by kocio-pl** [comment]
Water well:
3) ![water_well-3](https://user-images.githubusercontent.com/5439713/35239078-b2f310a6-ffaf-11e7-8843-933e69ed7a50.png)
4) ![water_well-4](https://user-images.githubusercontent.com/5439713/35239085-b92552fe-ffaf-11e7-866f-be39bd589b67.png)

**Comment by kocio-pl** [comment]
Pump:
5) ![pump-5](https://user-images.githubusercontent.com/5439713/35240174-e5b42bc6-ffb2-11e7-87a3-9e9b18d1d302.png)
6) ![pump-6](https://user-images.githubusercontent.com/5439713/35240183-ec5c44cc-ffb2-11e7-9874-6825537ce300.png)

**Comment by Tomasz-W** [comment]
Water well: honestly, both icons are good enought, we can treat "3)" as little bit more readable, because it has less elements
Pump: I can see there is thicker line on the right side in "6)", but also, both icons are good enought, but I would choose "5)"

**Comment by dieterdreist** [comment]
2018-01-23 9:08 GMT+01:00 Tomasz-W <notifications@github.com>:

> Water well: honestly, both icons are good enought, we can treat "3)" as
> little bit more readable, because it has less elements
> Pump: I can see there is thicker line on the right side in "6)", but also,
> both icons are good enought, but I would choose "5)"
>


while I find the pump well readable, it might be missleading, because a
water well is simply "a facility created to access ground water from an
aquifer." i.e. it is a shaft or a duct, a pump is an additional feature
that can be present or not.

**Comment by kocio-pl** [comment]
Water well icon is meant for general cases, pump icon is for https://wiki.openstreetmap.org/wiki/Key:pump (subcase of a water well).

**Comment by Tomasz-W** [comment]
@kocio-pl Could you publish Gist link with SVGs?

**Comment by kocio-pl** [comment]
Sure, here they are:

https://gist.github.com/kocio-pl/d915ce4049c49d6f52f1fd765a486fe0

**Comment by jeisenbe** [comment]
@Imagico proposed a very stylized icon for water wells which would be similar to his new spring design. The images also show similar stylized icons for water taps, hot springs, fountains and geysers:

z14-z16
![](http://blog.imagico.de/wp-content/uploads/2018/06/water-points_z14-z16.png)

z17
![](http://blog.imagico.de/wp-content/uploads/2018/06/water-points_z17.png)

z18 
![](http://blog.imagico.de/wp-content/uploads/2018/06/water-points_z18.png)

I'd be willing to make some test images with these icons as well as those above, if we can decide what shade of blue to use for wells, springs and fountains.

**Comment by jeisenbe** [comment]
I wasn’t suggesting making all of those changes, but just trying the icons
for water wells, springs, hot springs and geysers, especially those last 3.

I don’t think we need to change the others, since we are using blue for
transportation as accommodation POIs.

What color were you thinking for the well icon? Amenity brown or man_made
instead of blue?

**Comment by kocio-pl** [comment]
> I wasn’t suggesting making all of those changes, but just trying the icons for water wells, springs, hot springs and geysers, especially those last 3.

I thought that is your intention, because you posted it in this ticket.

> What color were you thinking for the well icon? Amenity brown or man_made instead of blue?

Most probably brown.

**Comment by Tomasz-W** [comment]
@jeisenbe Anyway, if you plan some test renderings of water wells, please include also these icons: https://gist.github.com/kocio-pl/d915ce4049c49d6f52f1fd765a486fe0 in _amenity-brown_ and _man-made-grey_ versions.

**Comment by dieterdreist** [comment]
sent from a phone

> On 21. Oct 2018, at 06:33, jeisenbe <notifications@github.com> wrote:
> 
> I wasn’t suggesting making all of those changes, but just trying the icons
> for water wells, springs, hot springs and geysers, especially those last 3.


I do suggest to adopt Christoph‘s  scheme for water sources, for me it is working great: from a superficial glance you see blue (or red) dots, and at closer inspection there is a typology encoded in fine details, such as the orientation of the point icon according to the way direction, it is subtle and shows the richness of the data in an unobtrusive way.

If you believe there is not enough contrast then the color could be tweaked.

**Comment by jeisenbe** [comment]
**TL:DR** - Water wells are important in developing countries and very rural areas, usually not important in western Cities. Different rendering for drinking water yes/no and pump manual/powered/no is a good idea; see Humanitarian style icons below.
----------------------
I've researched this issue, and there is a wide variation in significance and appearance of water wells. This is a feature that is of low importance in European cities but high importance in other contexts. 

https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dwater_well
The wiki page suggests use of drinking_water=yes/no and 
pump=* - How the water is drawn... (pump=powered,  pump=manual, pump=no), which may help. Out of 100,000 wells there are 16,000 with pump=* and 14,000 with drinking_water
https://taginfo.openstreetmap.org/tags/man_made=water_well#combinations

The "Humanitarian" HDM style shows four different icons for hand-draw wells, manual pumps, mechanized pumps, and wells that provide drinking water. @Imagico's alt colors style shows a cup icon with water wells and springs that supply drinking water, which is a nice idea.

Humanitarian icons: 
https://github.com/hotosm/HDM-CartoCSS/blob/master/icons/poi/bucket.svg
https://github.com/hotosm/HDM-CartoCSS/blob/master/icons/poi/pump_manual.svg
https://github.com/hotosm/HDM-CartoCSS/blob/master/icons/poi/pump_powered.svg
https://github.com/hotosm/HDM-CartoCSS/blob/master/icons/poi/drinking_water_well.svg

In Europe, there are a still a number of cute "medieval" style wells, complete with stone or brick wall, roof, bucket, rope and hand crank. There are also a number of early-modern pump wells, for example in Berlin there are 2100 "Berliner Straßenbrunnen" according to this page, https://wiki.openstreetmap.org/wiki/DE:Berliner_Straßenbrunnen , and over 1000 are mapped.

However, in the whole of the UK only 327 water wells are tag, because these features are no longer very important in Europe, where drinking water is universally available from every tap. http://taginfo.openstreetmap.org.uk/tags/man_made=water_well#overview

In contrast, water well are still extremely important in the developing world, especially in Africa and South Asia, where piped water supplies are still uncommon. Liberia has over 4000 water wells mapped, even though it only has 10MB of data, vs over 1 GB for the UK, suggesting that water wells are much more important in West Africa vs Great Britain. Haiti has almost 1500, 

Often the well will be at the center of a village or hamlet. Many are still hand-dug wells, with water drawn by bucket, though they lack the cute medieval Euro-style roof and crank. But manual pumps are more common, especially in areas where the water table is low, so wells are now bored by machine. Some pumps are mechanized. Not all of these well provided safe drinking water. Especially hand-dug, open wells are at risk for contamination, but some bored or drilled wells with pumps provide safe drinking water.

Another consideration is that many people have private wells, with an electric or manual pump, meant only for the use of one or two families. For example, my Indonesian neighbors and I share a hand-dug well with an electric pump, and the new shop/house next store just bored a well for their exclusive use. And back home in the USA, many rural communities rely on private wells for each house, with an electric pump to provide piped water to the house. Fortunately, very few of these private wells are currently mapped, but the micro-mapping trend may be a problem in the future if there is no access restriction rendered.

The other areas with wells are drylands farming and ranching lands in developed countries, for example the North American West and Australian Outback, where a bored well may be the only feature for kilometers around. Some of these have a modern wind-powered pump, man_made=windpump, others have solar-powered pumps or are connected the electric grid in less remote places. In Northern Australia there are dozens of bored wells with unique names, which apparently are local landmarks (because one mapper was tempted to add place=locality to get them to render)

**Comment by kocio-pl** [comment]
Thanks for your research, especially for the unique first hand evidence from Asia. This is basically what I have imagined and I'd like to see the rendering soon. Would you like to prepare a PR for that?

**Comment by jeisenbe** [comment]
First color tests. 

I'd suggest using an abstract symbol or a bucket or a manual pump for the default man_made=well icon. I like blue the best for these, however. If we have to use man-made gray or amenity-brown, I might go with the manual pump icon for default wells. I would use the "old fashioned" icon only for wells with an explicit "pump=no" tag, if at all. 

I'm not thrilled with the powered pump icon that I borrowed from the HDM (humanitarian) style. Perhaps this could be improved?

_Drilled water well in the outback of Australia, near an intermittent stream, but no other distractions:_

**Water blue:**
Abstract
![snake-imagico-blue](https://user-images.githubusercontent.com/42757252/48261263-d4537500-e461-11e8-85d8-f9f2c7b64696.png)
Bucket
![snake-bucket-blue](https://user-images.githubusercontent.com/42757252/48261274-dd444680-e461-11e8-96ea-30fa5ca4aae9.png)
Old-fashioned
![snake-old-blue](https://user-images.githubusercontent.com/42757252/48261262-d3bade80-e461-11e8-85ad-62efb4a04d5b.png)
Pump-5
![snake-pump-blue](https://user-images.githubusercontent.com/42757252/48261271-d9b0bf80-e461-11e8-82b9-39cbadf7ccba.png)
Pump-HDM
![snake-pump-hdm-blue](https://user-images.githubusercontent.com/42757252/48261266-d6b5cf00-e461-11e8-8624-a602a1a2edd7.png)
Powered Pump (HDM)
![snake-powered-blue](https://user-images.githubusercontent.com/42757252/48261254-cdc4fd80-e461-11e8-8f19-49d2a555389c.png)
Drinking Water Well (HDM)
![snake-drinking-blue](https://user-images.githubusercontent.com/42757252/48261257-d1588480-e461-11e8-99e9-da42157dd059.png)

**Man-Made Gray:**
![snake-old-gray](https://user-images.githubusercontent.com/42757252/48261200-9d7d5f00-e461-11e8-88d0-68330ede0c87.png)
![snake-bucket-gray](https://user-images.githubusercontent.com/42757252/48261222-b2f28900-e461-11e8-8ee0-a26c69d8d380.png)
![snake-pump-hdm-gray](https://user-images.githubusercontent.com/42757252/48261225-b6861000-e461-11e8-8bf5-a657fa29b9e2.png)
![snake-pump-gray](https://user-images.githubusercontent.com/42757252/48261231-bb4ac400-e461-11e8-935f-75273f5f512b.png)
![snake-powered-gray](https://user-images.githubusercontent.com/42757252/48261350-1aa8d400-e462-11e8-887c-695867f90619.png)

**Amenity Brown:**
Bucket
![snake-bucket-brown](https://user-images.githubusercontent.com/42757252/48261566-dd911180-e462-11e8-88bf-24e6a6880203.png)
Old-fashioned
![snake-old-brown](https://user-images.githubusercontent.com/42757252/48261335-0d8be500-e462-11e8-8b99-285d74d4f6ad.png)
Pump-5
![snake-pump5-brown](https://user-images.githubusercontent.com/42757252/48261599-f8638600-e462-11e8-867a-96d9229eb0f6.png)
Manual Pump (HDM)
![snake-pump-hdm-brown](https://user-images.githubusercontent.com/42757252/48261733-5abc8680-e463-11e8-977d-7ec10f627702.png)
Powered Pump (HDM)
![snake-powered-brown](https://user-images.githubusercontent.com/42757252/48261917-f64df700-e463-11e8-84be-f5d5236eaa0b.png)
Drinking water well (HDM)
![snake-drinking-brown](https://user-images.githubusercontent.com/42757252/48261985-3a40fc00-e464-11e8-9dff-1c70ecb41c43.png)

**Comment by kocio-pl** [comment]
My first impressions:
- for me brown works the best 
- I prefer Manual Pump (HDM) over Pump-5
- abstract does not work

**Comment by Tomasz-W** [comment]
@jeisenbe Thanks for tests! 

* I like the idea of using medieval well icon only for combination with `pump=no `
* I would use `man-made-grey` because `amenity-brown` makes water wells looking much more important than they usually are
* for generic icon I would use 'Pump-5' ('Manual Pump (HDM)' looks too caricatural for me)
* for combination with `pump=no` I would use  'bucket' or 'old-fashionated'
* 'Drinking water well (HDM)' icon may be a very good candidate for using it in https://github.com/gravitystorm/openstreetmap-carto/issues/3011

**Comment by jeisenbe** [comment]
Tests in Liberia and Berlin, first with improved abstract rendering. Before I had tried changing the color of the icon, but this caused it to render poorly. The abstract icon is the best option for rendering at lower zoom levels (z16 and z17), I believe. It also makes it easy to show drinking_water=yes.

**Abstract**
Simple design with supplemental cup icon for drinking water wells at z18.
Smaller icon used at z16 to allow rendering at this level without clutter.
Same water icon color used for drinking water amenity for consistency

z16 Liberia (multiple types of pumps, wells and drinking water sources)
![liberia-z16-abstract](https://user-images.githubusercontent.com/42757252/48270232-e7c00980-e47c-11e8-8dad-92cfc3c94e15.png)
z17
![liberia-z17-abstract](https://user-images.githubusercontent.com/42757252/48270246-ee4e8100-e47c-11e8-9a2e-00ad320a419b.png)
z18
![liberia-18-abs-drink](https://user-images.githubusercontent.com/42757252/48270255-f3abcb80-e47c-11e8-9f42-5c70d6dab1ab.png)
z19
![liberia-19-abs-drink](https://user-images.githubusercontent.com/42757252/48270263-f7d7e900-e47c-11e8-9ccf-6f9b8b4cd170.png)

z16 Liberia closed ways (Pumps, drinking water)
![ways-z16-abstract](https://user-images.githubusercontent.com/42757252/48270410-5735f900-e47d-11e8-84e6-51ade2607e33.png)
z17
![ways-z17-abstract](https://user-images.githubusercontent.com/42757252/48270395-4b4a3700-e47d-11e8-9a01-35b297175d28.png)
z18
![ways-z18-abs-dirnk](https://user-images.githubusercontent.com/42757252/48270311-1b029880-e47d-11e8-8f8b-96f27d0a1eed.png)

z16 Berlin historic Well
![berlin-z16-abstract](https://user-images.githubusercontent.com/42757252/48270342-3077c280-e47d-11e8-817d-0748b6b6239e.png)
z17
![berlin-z17-abstract](https://user-images.githubusercontent.com/42757252/48270347-353c7680-e47d-11e8-910f-b8d0993548cd.png)
z18
![berlin-z18-abs](https://user-images.githubusercontent.com/42757252/48270444-716fd700-e47d-11e8-9589-2067c2f62c33.png)

Berlin Pump
z17
![berliner-z17-abstract](https://user-images.githubusercontent.com/42757252/48270502-995f3a80-e47d-11e8-974a-7c03af0bf8be.png)
z19
![berlin-z19-abstract](https://user-images.githubusercontent.com/42757252/48270503-9a906780-e47d-11e8-83a2-ee8cfd89b720.png)

**Comment by daganzdaanda** [comment]
Thanks for all the examples! 
IMHO the abstract symbols work best. I do also like blue as a very intuitive water feature color. I'm not sure if there would be a conflict with transportation, tourism or office colours. The small size of the abstract symbols may be enough to avoid a real conflict. 
Because of their size and complex shape, the more literal symbols are maybe too "loud".
Of course the abstract symbols have to be "learned" maybe a bit more than a "realistic" icon.

**Comment by Tomasz-W** [comment]
Water-blue colour might be confusing as water well can be an abandoned/ turned off free standing object.

**Comment by jeisenbe** [comment]
The abstract rendering is good for the basic sense of “water well”, which
literally means “hole in the ground that provides access to ground water”.

I also like how the small abstract icon looks on z16; it is still visible
in rural areas and developing countries but does not add “clutter” in
well-mapped cities. Christopher suggested rendering it at z15 too, but z16
is probably enough.

It will be intuitive if we use a similar color and style for springs, as I
recommended in that issue.

Re: “water well can be an abandoned/ turned off free standing object.”

Shouldn’t abandoned features be tagged differently?

I believe that only public, usable water wells should be rendered at most
zoom levels.

(Perhaps a broken or private pump could be rendered at z19 with man-made
gray as a minor orientation point)

**Comment by Tomasz-W** [comment]
Abstract shape is totally not understandable also for me.

**Comment by jeisenbe** [comment]
Perhaps @Tomasz-W can design a pump that is a compromise between pump-5 and the HDM manual pump?

**HDM Manual Pump icon in Blue**
Liberia z17
![liberia-manual-hdm-z17-blue](https://user-images.githubusercontent.com/42757252/48301694-2b764a00-e535-11e8-898a-c68929b1ef66.png)
Liberia z18
![liberia-manual-hdm-blue-z18](https://user-images.githubusercontent.com/42757252/48301695-2dd8a400-e535-11e8-9881-15f5be80812a.png)
Liberia z19
![liberia-manualhdm-blue-z19](https://user-images.githubusercontent.com/42757252/48301698-2fa26780-e535-11e8-9499-f1245c3957e8.png)
Berliner z17
![berliner-hdm-z17-blue](https://user-images.githubusercontent.com/42757252/48301703-5496da80-e535-11e8-973d-96c739aee004.png)
Berliner z19
![berliner-manual-hdm-blue-z19](https://user-images.githubusercontent.com/42757252/48301704-595b8e80-e535-11e8-9f44-20b234387ab4.png)
Liberia 2
![liberia2-hdm-blue-z18](https://user-images.githubusercontent.com/42757252/48301706-64aeba00-e535-11e8-8fdc-4530346f852c.png)
Japan Temple z18
![japantemple-z18-circle](https://user-images.githubusercontent.com/42757252/48301739-ec94c400-e535-11e8-8aef-c855ad68cc2e.png)


**HDM Manual Pump icon in Brown**
Liberia z17
![liberia-hdm-brown-z17](https://user-images.githubusercontent.com/42757252/48301719-a8092880-e535-11e8-849e-a046a40feb7c.png)
Liberia z19
![liberia-hdm-brown-z19](https://user-images.githubusercontent.com/42757252/48301720-accddc80-e535-11e8-96a3-bafb27930cc4.png)
Berliner z19
![berliner-brown-hdm-z17](https://user-images.githubusercontent.com/42757252/48301722-b1929080-e535-11e8-9874-f88692943f79.png)
Liberia 2
![liberia2-hdm-brown](https://user-images.githubusercontent.com/42757252/48301725-b9523500-e535-11e8-9a69-73bb778243e8.png)
Japan z19
![japantemple-z19-pump-brown](https://user-images.githubusercontent.com/42757252/48301729-c2db9d00-e535-11e8-9e32-cb9cbfea98da.png)

**HDM Manual Pump icon in Gray**
Liberia z17
![liberia-z17-hdm-gray](https://user-images.githubusercontent.com/42757252/48301742-fc140d00-e535-11e8-87d9-9382a30012d0.png)
Liberia z19
![liberia-hdm-gray-z19](https://user-images.githubusercontent.com/42757252/48301744-fd453a00-e535-11e8-9750-4e79233f6ab3.png)
Berliner 
![berliner-hdm-gray](https://user-images.githubusercontent.com/42757252/48301745-ffa79400-e535-11e8-9852-1650b81d2d80.png)
Liberia 2
![liberia2-hdm-gray](https://user-images.githubusercontent.com/42757252/48301746-00d8c100-e536-11e8-9015-c793488d4136.png)
Japan Castle z18
![japan-castle-hdm-gray-z18](https://user-images.githubusercontent.com/42757252/48301747-046c4800-e536-11e8-9272-2d7ac1bf1b6d.png)
Japan Temple z18
![japan-temple-hdm-gray-z18](https://user-images.githubusercontent.com/42757252/48301749-0504de80-e536-11e8-97c4-196316500d62.png)

**Comment by Tomasz-W** [comment]
As I very like dot-rendering for some features on z17, because it's the simplest possible shape, I consider different icons for the same feature at different zoom level as bad idea:
* as @Adamant36 said, "it might also give the false impression that there is another mapped poi next to the main one"
* in might give some bug or rendering-in-progress impression
* proposed smaller icon is not intuitive at all

**Comment by sfkeller** [comment]
There are currently >111 K objects tagged man_made=water_well in OSM.  And it's mentioned in https://wiki.openstreetmap.org/wiki/Map_Features .

Finally I just want to point to the fact, that the lack of a water well  icon in Carto style obviously leads to "tagging for the renderer", where mappers just tag water wells as amenity=fountain or drinking_water.

**Comment by jragusa** [comment]
Owing to the large abundance of this tag, why not using three distinct icons with regard to the tag `pump` (https://wiki.openstreetmap.org/wiki/Key:pump) which is also frequently used (26 799) if we only consider recommended values:
* `pump=no`: old fashion well
* `pump=yes`: Pump HDM
* `pump=powered`: Powered pump HDM

**Comment by jragusa** [comment]
For the record, there is a proposal related to pump features : https://wiki.openstreetmap.org/wiki/Proposed_features/Pumping_proposal

This also applies to `man_made=windpump` (#4149) and `man_made=petroleum_well` (#3494)

**Comment by mfbehrens99** [comment]
Is anybody still working on this topic?

**Comment by imagico** [comment]
Speaking for myself only - i have on my todo list the idea to further explore the concept of compound symbols (for amenity=drinking_water/drinking_water=yes in combination with man_made=water_well/man_made=water_tap/natural=spring but also for things like amenity=watering_place - see https://github.com/gravitystorm/openstreetmap-carto/issues/3703#issuecomment-469659265, https://github.com/gravitystorm/openstreetmap-carto/issues/4441#issuecomment-882059005)

Someone working on a suitable implementation of rendering man_made=water_well would definitely be helpful.  But as you can see above there is no consensus among the maintainers how this should look like.  I would not mind a solution different from my implementation but it would have to be non-misleading (in particular regarding confusion with man_made=water_tap - see #3011) and culturally generic so it is suitable for all kinds of wells in different settings.

The solution i showed (http://blog.imagico.de/the-way-of-the-water/) has in particular the advantages of

* offering a solution for natural=spring, natural=geyser and natural=hot_spring as well (#4073) - and fountains (#1934)
* being suitable for a subtle rendering starting at an early zoom level.
* being suitable for compound symbols illustrating additional properties.
* not implying a specific type of well or a specific mechanism of water extraction.

**Comment by sfkeller** [comment]
This is just for your information in support of this proposal - especially imagico's - in addition to my arguments here https://github.com/gravitystorm/openstreetmap-carto/issues/1224#issuecomment-616223832 .

Currently there's a proposed tag man_made=borehole [1] and the Project of the month Switzerland of October 2021 [2] will be "Fountains" including amenity=fountain and others like amenity=drinking_water / water_point and man_made=water_well / water_tap. 

I know well the burden of volunteered maintenance. But the fact that only amenity=fountain and amenity=drinking_water (from the above list) are rendered in OSM carto style - although proposed since 6 years - means, that we need additional energy now, motivating mappers and preventing newcomers not to map for the renderer.

[1] https://wiki.openstreetmap.org/wiki/Proposed_features/Bore
[2] https://wiki.openstreetmap.org/wiki/PotM_CH

**Comment by BertMule** [comment]
My use is both in medieval setting as a true well (like [Burcht](https://www.openstreetmap.org/edit#map=20/52.15894/4.49244)), and as a pump in historic setting like on squares or community gardens ([Sint Annahofje.](https://www.openstreetmap.org/edit#map=19/52.15669/4.49624))
I can imagine to distinguish with separate icons. 

This topic is around since 2015! 
It's disappointing that though we agree there should be something, and many proposale have been made, this topic is already left behind for a 8 years.
I'd rather see a tag removed/replaced, than that it is effectively useless.

There are a lot more of them.
I would say that every accepted tag (no too many please) should also be rendered (at least on some map).

**Comment by sfkeller** [comment]
> It's disappointing that though we agree there should be something, and many proposale have been made, this topic is already left behind for a 8 years.

Agreed. But now we are so close. And hope springs eternal!
